Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1176,7 +1176,7 @@ def audit_line(line, _lineno)
       problem "Use `assert_match` instead of `assert ...include?`"
     end
 
-    if line.include?('system "npm", "install"') && !line.include?("Language::Node") && formula.name !~ /^kibana(\d{2})?$/
+    if line.include?('system "npm", "install"') && !line.include?("Language::Node") && formula.name !~ /^kibana(\@\d{1,2}(\.\d{1,2})?)?$/
```

Comment:
A simpler regex like `/^kibana(\@\d+(\.\d+)?)?$/` would suffice (and handle weird versions in future). Could you also add a newline after the final `&&` to avoid this line being more than 80 characters long? I appreciate it was already but: may as well clean this up as you're here 😁